hlapipeline/alignimages.py

```diff
@@ -724,8 +728,13 @@ def generate_astrometric_catalog(imglist, **pars):
 
     """
     # generate catalog
+    temp_pars = pars.copy()
+    if pars['output'] == True:
+        pars['output'] = 'ref_cat.ecsv'
+    else:
+        pars['output'] = None
     out_catalog = amutils.create_astrometric_catalog(imglist,**pars)
-
+    pars = temp_pars.copy()
     # if the catalog has contents, write the catalog to ascii text file
```

Comment:
Am I mistaken because I am a Python newbie?  Even though create_astrometric_catalog() did not write the table to file on disk, the table still exists in memory.  This table is passed back to the calling routine as out_catalog, and out_catalog may well have a length > 0.  In this case, we also should not write 'refcatalog.cat' to disk.

Reply:
Issue resolved in commit [e768560](https://github.com/spacetelescope/hlapipeline/pull/75/commits/e768560040e6517bd7460b14b84f9f672006d951). File 'refcatalog.cat' is now not written to disk if alignimages.perform_align() input parameter _**output**_=False.